### ISSUE TITLE
Cannot read property 'body' of undefined

### ISSUE THREAD

**Issue by abhijitkane** [balderdashy/sails.io.js#52]
We're seeing 'Cannot read property 'body' of undefined' in sails.io.js (Line 354), here:

```
// ( === the request method, e.g. 'get', 'post', 'put', etc. )
var sailsEndpoint = requestCtx.method;
socket.emit(sailsEndpoint, requestCtx, function serverResponded(responseCtx) {

// Adds backwards-compatibility for 0.9.x projects
// If `responseCtx.body` does not exist, the entire
// `responseCtx` object must actually be the `body`.
var body;
if (!responseCtx.body) {   << HERE
    body = responseCtx;
} else {
    body = responseCtx.body;
}
// Send back (emulatedHTTPBody, jsonWebSocketResponse)
```

The file begins with `/*! Socket.IO.min.js build:0.9.16,`
Is there anything else I can do to identify the version of the sails.io.js file I'm using?

I'm not able to reproduce the exact steps on the client that cause this. This error was found in the error log a few times.

**Comment by sailsbot** [comment]
Thanks for posting, @abhijitkane.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
